netket/vqs/mc/mc_state/state.py

```diff
@@ -636,7 +652,7 @@ def expect_and_forces(
         if mutable is None:
             mutable = self.mutable
 
-        return expect_and_forces(self, Ô, self.chunk_size, mutable=mutable)
+        return expect_and_forces(self, Ô, self.chunk_size, self.grad_chunk_size, mutable=mutable)
```

Comment:
Doesn't it make more sense to just do 

```
     chunk_size = self.grad_chunk_size
     if chunk_size is None:
           chunk_size = self.chunk_size
     return expect_and_forces(self, Ô, self.chunk_size, mutable=mutable)
```

? 

That way you don't need to touch everything else, because in any case there is just one chunk size being used in those underlying functions.

Reply:
The function `forces_expect_hermitian_chunked` uses both arguments. `chunk_size` to get the local energies, and `grad_chunk_size` to get the forces.

Reply:
ah, right. ok then